README.md

```diff
@@ -28,6 +28,163 @@ Plug 'tribela/vim-transparent'
 [Vim-Plug]: https://github.com/junegunn/vim-plug
 
 
+## Configuration
+
+### Minimal configuration
+
+```vim
+Plug 'tribela/vim-transparent'
+
+" Clear the background of highlight groups in g:transparent_groups.
+let g:transparent_groups = ['Normal', 'Comment', 'Constant', 'Special', 'Identifier',
+                            \ 'Statement', 'PreProc', 'Type', 'Underlined', 'Todo', 'String',
+                            \ 'Function', 'Conditional', 'Repeat', 'Operator', 'Structure',
+                            \ 'LineNr', 'NonText', 'SignColumn', 'CursorLineNr', 'EndOfBuffer']
```

Comment:
As default values are set, This should not be set on minimal configuration
```suggestion
```

Reply:
I have deleted the `Minimal Configuration` and commited.